libcloud/compute/drivers/ec2.py

```diff
@@ -5545,6 +5586,15 @@ class EC2EUNodeDriver(EC2NodeDriver):
     _region = 'eu-west-1'
 
 
+class EC2EUCentralNodeDriver(EC2NodeDriver):
```

Comment:
I will probably do something along those lines:

1. Modify `SignedAWSConnection` class to take `version` argument in the constructor and default it to `2`.
2. Modify `_ex_connection_class_kwargs` method to pass the correct version to the connection class constructor in case the region is `eu-central-1`
3. Remove the new `EC2EUCentralNodeDriver``class since the class based approach has been deprecated quite a while ago now.

@gertjanol Will change nr. 3 affect you? Do you use a class per region approach?